html/includes/common/alerts.inc.php

```diff
@@ -291,7 +291,8 @@
             success: function(msg){
                 $("#message").html(\'<div class="alert alert-info">\'+msg+\'</div>\');
```

Comment:
This slightly becomes an issue as it never disappears so if you can change it to:

`toastr.success(msg);`

Then it should show the toastr pop and go after a few seconds